priv/repo/datasets.json

```diff
@@ -17,11 +17,6 @@
     "slug": "arrets-parcours-et-horaires-theoriques-du-reseau-irigo-gtfs",
     "coordinates": [-0.556585, 47.47207],
     "catalogue_id": "vwti_qvcukzprkegeyyffrfsqpu"
-}, {
-    "spatial": "M\u00e9tropole europ\u00e9enne de Lille",
-    "coordinates": [3.05644, 50.631564],
-    "slug": "gtfs-et-visualisation-du-reseau-transpole",
```

Comment:
Why remove transpole?
It was fixed in https://github.com/etalab/transport-site/pull/254/files#diff-7e38887a63008e17469c175d57591c15L23